javascript/javascript1/week1/dog_age_calculator.js

```diff
@@ -0,0 +1,14 @@
+// Goodboy-Oldboy (A dog age calculator)
```

Comment:
the formula to calculate dog age in human years:
1 dog year = 7 human years
10 years old = 70 human years